utils/wesnoth-map-diff/src/images.ts

```diff
@@ -17,11 +17,15 @@ const getDictTerrainType2ImagesPath = async () => {
   const dictTerrainType2ImagesPath = rootByTagName(terrain)
     .terrain_type
     .reduce((acc, terrainType) => {
-      if (terrainType.symbol_image === undefined) {
-        return acc
+      if (terrainType.symbol_image) {
+        acc[terrainType.string.value] = terrainType.symbol_image.value
+      }
+      if (!acc[terrainType.string.value] && terrainType.editor_image) {
+        acc[terrainType.string.value] = terrainType.editor_image.value 
+      }
+      if (!acc[terrainType.string.value]) {
```

Comment:
There should probably be a message here so we can see what is missing.

Reply:
how would I do that? @macabeus help?

Reply:
You can use `console.log` to output a message in the console - that will be displayed in GitHub Actions.

Reply:
For instance:
```
console.log(`No image defined for the tile "${terrainType.string.value}"`)
```

Reply:
added in [32be9d8](https://github.com/wesnoth/wesnoth/pull/7452/commits/32be9d85576193101aa6cf50b095b4125b615429)